workflows/ConfiguredQuery/content.json

```diff
@@ -3626,11 +3659,13 @@
             "id": 730
         }
     ],
-    "licenseInfo": { "licenseeId": "88cb61d6-af47-9b3e-5181-ed674c4c0815" },
+    "licenseInfo": {
+        "licenseUrl": "https://vn-demo.vertigis.com/vertigisstudio/workflow/service/auth/license"

```

Comment:
We should remove the `licenseInfo`

Reply:
If it is in the other workflow please remove it there too.

Reply:
Fixed